evap/staff/views.py

```diff
@@ -489,6 +489,23 @@ def semester_todo(request, semester_id):
     template_data = dict(semester=semester, responsible_list=responsible_list)
     return render(request, "staff_semester_todo.html", template_data)
 
+@staff_required
+def semester_grade_reminder(request, semester_id):
+    semester = get_object_or_404(Semester, id=semester_id)
+
+    courses = semester.course_set.filter(state__in=['evaluated', 'reviewed', 'published'], is_graded=True, gets_no_grade_documents=False).all().prefetch_related("degrees")
```

Comment:
you can remove the `prefetch_related`-part, since you're not accessing the degrees of the courses.